docs/public-networks/concepts/sync-node.md

```diff
@@ -28,7 +28,45 @@ Archive nodes can do everything full nodes do, and they can also access historic
 This means that archive nodes require more disk space than full nodes.
 
 Besu must connect with other peers to sync with the network. If your node is having trouble peering, 
-try [troubleshooting peering](../../how-to/troubleshoot/peering.md).
+try [troubleshooting peering](../how-to/troubleshoot/peering.md).
+
+Besu supports several synchronization options for different network types and use cases:
+
+## Public network syncing
+
+For public networks, Besu offers the following sync modes:
+
+- [Snap synchronization](#snap-synchronization)
+- [Checkpoint synchronization](#checkpoint-synchronization)
+- [Fast synchronization](#fast-synchronization)
+- [Full synchronization](#run-an-archive-node)
+
+:::tip
+
+We recommend snap sync because it follows the Ethereum specification and enables you to serve full historical data.
+
+:::
+
+| Sync Mode | Description | Requirements | Method | Limitations |
+|-----------|-------------|--------------|--------|-------------|
+| Snap | Default for Mainnet. Efficient sync starting from the genesis block, the successor to Fast sync. | Besu version 22.4.0 or later | Downloads as many leaves of the trie as possible, reconstructs the trie locally. Snap is faster than Fast sync. | Cannot switch from fast sync to snap sync mid-process. |
+| Checkpoint | Efficient sync from a specific checkpoint block configured in the genesis file. | Besu version 22.4.3 or later | Syncs from a checkpoint block defined in the genesis file. It is the fastest sync mode and has the lowest storage requirements. | |
+| Fast | Default for named networks except the dev development network. | None | Downloads block headers and transaction receipts, verifies chain from genesis block. | Might become impossible to sync Ethereum Mainnet in the future. |
+| Full | Downloads and verifies the entire blockchain and state from the genesis block. This builds an archive node, with full state history.| None | Downloads entire blockchain, verifies all states from genesis block. | Slowest sync mode, requires the most disk space. |
```

Comment:
There seems to be some overlap across the "Method" and "Description" columns. Since this is just an overview table, I suggest making this it smaller by combining the two (maybe just "Description"). You can specify which modes are defaults for which networks in their respective extended sections.